src/Microsoft.ML.TimeSeries/TimeSeriesStatic.cs

```diff
@@ -0,0 +1,315 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.TimeSeriesProcessing;
+using Microsoft.ML.StaticPipe.Runtime;
+using System.Collections.Generic;
+
+namespace Microsoft.ML.StaticPipe
```

Comment:
>StaticPipe [](start = 23, length = 10)

TimeSeries has a separate nuget  Microsoft.ML.TimeSeries.

As such, the static extensions for time seies should be placed in a separate assembly.